docs/proposals/20240807-in-place-updates.md

```diff
@@ -886,7 +896,7 @@ Since the fallback to machine replacement is enabled, the MachineDeployment cont
 ##### `UpdateMachine` endpoint
 ##### Request
 > Requirements:
-> * Machine reference
+> *  Desired Machine Spec, including bootstrap and and infra machine templates
```

Comment:
why?
afaik by this point the machine object has already been updated with the desired spec